src/Domain/Analyser.php

```diff
@@ -105,6 +105,14 @@ private function analyseFile(Collector $collector, string $filename): void
 
                 if ($token === ';') {
                     if ($className !== null) {
+                        $blackList = ['{', '}'];
```

Comment:
I'm really not sure here, but what's happen if we have something like this : 

```php
$callable = function() { echo 'test';};

$callable();
```

Is it considered as a BadClass ? I should test against your branch, but if you know ? :wink: